__tests__/app.test.js

```diff
@@ -41,13 +75,37 @@ describe("/api/topics", () => {
           });
         });
     });
-    test("Status 404: responds with 'Path does not exist' message", () => {
+  });
+});
+
+describe("api/articles", () => {
+  describe("GET api/articles/:article_id", () => {
+    test("Status 200: responds with corresponding article object to requested id", () => {
+      return request(app)
+        .get("/api/articles/4")
+        .expect(200)
+        .then(({ body }) => {
+          //check id
+          expect(body.article_id).toBe(4);
+          //rest
+          expect(body).toHaveProperty("author", expect.any(String));
```

Comment:
Please remove comments and console.logs before committing.

Also, we can be more specific that expect.any() in these assertions, we know what these values should be and so lets check for them :)